zeppelin-web/src/app/credential/credential.controller.js

```diff
@@ -15,16 +15,31 @@
 'use strict';
 
 angular.module('zeppelinWebApp').controller('CredentialCtrl', function($scope, $route, $routeParams, $location, $rootScope,
-                                                                         $http, baseUrlSrv) {
+                                                                         $http, baseUrlSrv, ngToast) {
   $scope._ = _;
 
-  $scope.credentialEntity = '';
-  $scope.credentialUsername = '';
-  $scope.credentialPassword = '';
+  $scope.credentialInfo = [];
+  $scope.showAddNewCredentialInfo = false;
 
-  $scope.updateCredentials = function() {
-    if (_.isEmpty($scope.credentialEntity.trim()) ||
-        _.isEmpty($scope.credentialUsername.trim())) {
+  var getCredentialInfo = function() {
+    $http.get(baseUrlSrv.getRestApiBase()+'/credential').
+      success(function(data, status, headers, config) {
+        $scope.credentialInfo = data.body;
+        console.log('Success %o %o', status, data.message);
+      }).
+      error(function(data, status, headers, config) {
+        console.log('Error %o %o', status, data.message);
+      });
+  };
+
+  var credentialInfoTmp = [];
```

Comment:
This isn't used anywhere and the index are not set well anyway.
all the call to this variable should be removed